app/templates/index.hbs

```diff
@@ -22,7 +22,17 @@
         {{/header.dropdown}}
       {{/banner.header}}
 
-      {{component banner.details}}
+      {{#banner.details}}
```

Comment:
Could you include a note about why you made this template change? i.e. what problem does it solve

Reply:
oftentimes on demo the full metadata profile for pieces isn't present, ie the title of the piece shows up but not the composer, as there is a data dump that producers do each day that only makes it into production. So, maybe 1/3 or 1/4 of the time the banner on demo defaults to the show name, making it difficult to test how accurate the metadata alignment is on demo. This solves that issue by displaying the piece title even when the composer name isn't present

Reply:
great, thanks